docs/authentication.md

```diff
@@ -101,12 +101,30 @@ SQLPad users do not need to be added ahead of time, and may be created on the fl
 LDAP-based authentication can be enabled by setting the necessary environment variables:
 
 - `ENABLE_LDAP_AUTH`
-- `LDAP_URL`
+- `LDAP_URL` (support protocals: `ldap://` and `ldaps://`)
 - `LDAP_BASE_DN`
 - `LDAP_USERNAME`
 - `LDAP_PASSWORD`
+- `DISABLE_USERPASS_AUTH`=`false` (should enable local user logins)
 
-Users sign in to LDAP using a username (and not an e-mail address).
+or LDAP settings in INI format are as follows:
+
+```ini
```

Comment:
in this file I don't think you should write the JSON variable names, but only the ENV VARs. You should move the descriptions to the ENV VARs.